src/tests/ie_test_utils/common_test_utils/ngraph_test_utils.hpp

```diff
@@ -56,8 +56,10 @@ class TransformationTestsF : public  CommonTestUtils::TestsCommon {
             ASSERT_NO_THROW(check_rt_info(function));
         }
 
-        auto res = comparator.compare(function, function_ref);
-        ASSERT_TRUE(res.valid) << res.message;
+        if (!m_disable_function_comparison) {
+            auto res = comparator.compare(function, function_ref);
+            ASSERT_TRUE(res.valid) << res.message;
+        }
```

Comment:
@GlebKazantaev, it is workaround to make it possible to check accuracy on pruning transformation. Should be discussed